WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailListFragment.java

```diff
@@ -355,34 +356,35 @@ private boolean hasNoteBlockAdapter() {
             if (bodyArray != null && bodyArray.length() > 0) {
                 for (int i = 0; i < bodyArray.length(); i++) {
                     try {
-                        JSONObject noteObject = bodyArray.getJSONObject(i);
+                        FormattableContent noteObject = mNotificationsUtilsWrapper
+                                .mapJsonToFormattablbeContent(bodyArray.getJSONObject(i));
                         // Determine NoteBlock type and add it to the array
                         NoteBlock noteBlock;
-                        String noteBlockTypeString = JSONUtils.queryJSON(noteObject, "type", "");
 
-                        if (BlockType.fromString(noteBlockTypeString) == BlockType.USER) {
+                        if (BlockType.fromString(noteObject.getType()) == BlockType.USER) {
```

Comment:
Could we have this from the enum instead of a String?

Reply:
We have an enum just for the Range type, not for the FormattableContent type.